src/types/guilds/modifyGuildMember.ts

```diff
@@ -10,4 +10,6 @@ export interface ModifyGuildMember {
   deaf?: boolean | null;
   /** Id of channel to move user to (if they are connected to voice). Requires the `MOVE_MEMBERS` permission */
   channelId?: bigint | null;
+  /** when the user's timeout will expire and the user will be able to communicate in the guild again (up to 28 days in the future), set to null to remove timeout. Requires the `MODERATE_MEMBERS` permission */
+  communication_disabled_until?: number;
```

Comment:
```suggestion
  communicationDisabledUntil?: number;
```

Reply:
These are options that the user will provide, we do not want snake case from users.

Reply:
oops my bad too